.github/pull_request_template.md

```diff
@@ -7,6 +7,11 @@ For more information on the requirements for pull requests, please see [the CONT
 [] I confirm this contribution is my own and I agree to license it with Apache 2.0.
 [] I confirm the moderators may change the PR before merging it in.
 
+For new board enablements only.
+[] I confirm the board is Mbed enabled and passes the Mbed enabled test set.
+[] I confirm the contribution has been tested properly and tests results for [TESTS](https://github.com/ARMmbed/mbed-os-example-pelion/tree/master/TESTS) are attached.
+[] I confirm `mbed-os.lib` and `mbed-cloud-client.lib` hashes were and/or the content in those folders was not modified in order to pass the tests.
```

Comment:
Can we add an extra lines to help to visualize this?

Reply:
Added. GitHub shows these quite strangely.